ktor-client/ktor-client-features/ktor-client-json/ktor-client-serialization/common/src/io/ktor/client/features/json/serializer/KotlinxSerializer.kt

```diff
@@ -35,6 +27,7 @@ class KotlinxSerializer(
     /**
      * Set mapping from [type] to generated [KSerializer].
      */
+    @Deprecated("[setMapper] is obsolete with 1.3.50 `typeOf` feature", level = DeprecationLevel.WARNING)
     fun <T : Any> setMapper(type: KClass<T>, serializer: KSerializer<T>) {
         @Suppress("UNCHECKED_CAST")
         mappers[type as KClass<Any>] = serializer as KSerializer<Any>
```

Comment:
Do we actually need `mappers` and `listMappers`? We probably can keep these deprecated methods empty